HSStockChart/TimeLineChart/HSTimeLine.swift

```diff
@@ -107,6 +122,25 @@ public class HSTimeLine: UIView, HSDrawLayerProtocol {
         self.addGestureRecognizer(tapGesture)
     }
     
+    //Getter Setter for customization
+    public func setCountOfTimes(times: Int)
+    {
+        self.countOfTimes = times
+    }
+    
+    public func setfiveDayCount(counts: Int)
+    {
+        self.fiveDayCount = counts
+    }
+    
+    public func setDailyOpenTime(openTime: String, middleTime: String, closeTime: String)
+    {
+        self.openTime = openTime
+        self.middleTime = middleTime
+        self.closeTime = closeTime
+    }
+    
```

Comment:
You can change the `fileprivate`  for openTime, middleTime and closeTime.

Reply:
```suggestion
```